### PR TITLE
[JENKINS-38481] 1.11.37 changed com.amazonaws.services.ec2.AmazonEC2#createTags return type, so incompatible binary change

### PR DESCRIPTION
**Pull request by ndeloof**
Just need to recompile plugin against new API.
@reviewByBees

### CODE REVIEW

**Comment by daniel-beck** [comment]
FTR, outdated PR description, probably after a PR build complained?

---

@francisu Any chance this could be reviewed, merged and released in the not-too-distant future? Apparently, this issue is costing people quite a bit of money as you can see in comments to JENKINS-38481.

File: src/test/java/hudson/plugins/ec2/SlaveTemplateUnitTest.java [hunk 1]
```diff
@@ -45,8 +46,8 @@ public void tearDown() throws Exception {
     public void testUpdateRemoteTags() throws Exception {
         AmazonEC2 ec2 = new AmazonEC2Client() {
             @Override
-            public void createTags(com.amazonaws.services.ec2.model.CreateTagsRequest createTagsRequest) {
-
+            public CreateTagsResult createTags(com.amazonaws.services.ec2.model.CreateTagsRequest createTagsRequest) {
+                return null;
```
Comment (daniel-beck) [inline src/test/java/hudson/plugins/ec2/SlaveTemplateUnitTest.java]: I wonder whether #220 gets this right with `return new CreateTagsResult();`

**Comment by francisu** [comment]
@daniel-beck I will get this out today.

**Comment by francisu** [comment]
Thanks so much @ndeloof and @daniel-beck for taking care of this!

**Comment by daniel-beck** [comment]
@francisu teamwork \o/

**Pull request merged**